Program.cs

```diff
@@ -1,10 +1,181 @@
-﻿namespace Task1
+﻿using System;
+
+namespace Task1
 {
     internal class Program
     {
+        static void DisplayArray(int size, int[] array)
+        {
+            for (int i = 0; i < size; i++)
+            {
+                if (i < (size - 1))
+                {
+                    Console.Write($"{array[i]},");
+                }
+                else
+                {
+                    Console.Write($"{array[i]}]\n");
+                }
+            }
+        }
+
+        static bool ConvertToDuodecimal(int decimalNumber)
+        {
+            string duodecimal = "";
+            char duodecimalDigit;
+
+            while (decimalNumber > 0)
+            {
+                int remainder = decimalNumber % 12;
+
+                if(remainder < 10)
+                {
+                    duodecimalDigit = (char)(remainder + '0');
+                }
+                else
+                {
+                    duodecimalDigit = (char)(remainder - 10 + 'A');
+                }
+
+                duodecimal = duodecimalDigit + duodecimal;
+
```

Comment:
Usage of string methods (for example, Contains) is not allowed now )) Only in future

Reply:
Modified